src/main/java/gay/nyako/nyakomod/mixin/PlayerEntityMixin.java

```diff
@@ -130,6 +130,10 @@ public final ActionResult interact(PlayerEntity player, Hand hand) {
 
         ItemStack itemStack = player.getStackInHand(hand);
         if (itemStack.isOf(Items.BUCKET)) {
+            if (this.isBlocking()) {
+                this.playSound(SoundEvents.ITEM_SHIELD_BLOCK, 1.0F, 1.0F);
+                return ActionResult.success(this.world.isClient);
```

Comment:
I can only drink lactose free milk.

Reply:
If you keep that up, you won't be able to drink milk at all